apps/src/studio/skins.js

```diff
@@ -619,16 +619,37 @@ function loadHoc2015(skin, assetUrl) {
 
   skin.customObstacleZones = {endor : {}, hoth: {}, starship: {}};
 
+  function generateGridObstacleZones() {
+    var zones = [];
+    for (var row = 0; row < 4; row++) {
+      for (var col = 0; col < 4; col++) {
+        var zone = {
+          minX: 50 + col * 100 + 5,
+          maxX: 50 + col * 100 + 49 - 5,
+          minY: 50 + row * 100 + 15,
```

Comment:
Yes, I want these collision regions to be at the bottoms of the tiles, so that feet collide with the blocks' bases.  Of course it's not super readable, but better than a lot more raw numbers.